### DISCUSSION TITLE
Potential new elevator pitch / tagline for Datasette: The best way to publish data online

### DISCUSSION THREAD

**Discussion by simonw** [simonw/datasette#990] [General]
One of the biggest challenges I've had with Datasette is figuring out a good elevator pitch / tagline. The software does a lot of things, and I've found it really difficult to summarize that down to something pithy that helps people understand what it does.

I'm contemplating this: **The best way to publish data online**

It's a very bold statement, but I think it holds up. I honestly can't think of a better way of publishing structured data online, from the point of view of a consumer of data APIs - at least for the size of data usually used with it (if you have TBs of data you may need something else) - especially given the existence of plugins like [datasette-graphql](https://github.com/simonw/datasette-graphql).

If anyone can think of a better format or system for publishing data I'd love to hear about it! I don't feel comfortable claiming that it's "the best" unless I'm personally convinced that it's true.

As a tagline, this ignores a huge amount of work I've been doing recently in the Datasette ecosystem around modifying and collaborating on data - thinks like [datasette-upload-csvs](https://github.com/simonw/datasette-upload-csvs) and [datasette-edit-schema](https://github.com/simonw/datasette-edit-schema).

I think I'm OK with that. Datasette isn't (yet) the best tool for modifying and collaborating on data - things like Google Sheets offer much more powerful functionality on that front. But I'm comfortable stating that Datasette's unique selling point is as a way of publishing data such that other people can make use of it as quickly and easily as possible.

I would love to hear your feedback, suggestions, challenges and alternatives!

**Comment by simonw** [comment]
See also this Twitter thread: https://twitter.com/simonw/status/1312543663113682944

**Comment by simonw** [comment]
Jon Udell [on Twitter](https://twitter.com/judell/status/1312547893568364544) suggested dropping "online", which makes:

**The best way to publish data**

I think that's better. Omit needless words.

**Comment by simonw** [comment]
Alternative: **The better way to publish data**

**Comment by rgieseke** [comment]
Datasette *is* great but why have "best" or "better", you're not selling anything, right?
There is lots of data, e.g. NetCDF files or heavily nested, structured JSON data where it's not the best ...

How about "The fast way to publish data"? Or maybe "simple"?

Or how about "A better way to publish data" - same direction (and justified statement) but less competetive?

**Reply by simonw** [reply]
I'm definitely selling - I'm trying to grow an entire ecosystem around this, which means I need to pull in users, plugin authors and convince project stakeholders to make bets on the software. Selling isn't very natural to me though, which is why I'm trying to push myself to take on a more assertive marketing stance.

Personally I do like "a better way" more than "the better way", but I'm worried that I'm letting my own insecurities dampen the message if I do that.

**Reply by rgieseke** [reply]
I think for convincing the details and examples below (see e.g. Svelte, React or Django landing pages) are much more important that the first tag line, something impossible to prove ("best") might actually turn off some of the likely highly analytical target group.
